### PR TITLE
Aria Operations Rebranded Release, v1.8.14

### PR DESCRIPTION
**Pull request by ustinov**
Please review the updated `README.md` in a human-readable form here
https://github.com/wavefrontHQ/wavefront-sdk-python/tree/austinov/rebranding

### CODE REVIEW

**Review by oppegard — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -17,22 +18,22 @@
 * [How to Get Support and Contribute](#how-to-get-support-and-contribute)
 * [How to Release](#how-to-release)
 
-# Welcome to the Wavefront Python SDK
+# Welcome to VMware Aria Operations™ for Applications Python SDK
 
-Wavefront by VMware Python SDK lets you send raw data from your Python application to Wavefront using a `wavefront_sender` interface. The data is then stored as metrics, histograms, and trace data. This SDK is also referred to as the Wavefront Sender SDK for Python. 
+VMware Aria Operations for Applications (formerly known as Wavefront) Python SDK lets you send raw data from your Python application to Aria Operations for Applications using a `wavefront_sender` interface. The data is then stored as metrics, histograms, and trace data. This SDK is also referred to as the Wavefront Sender SDK for Python.
```
Comment (Margarita-Staneva) [inline README.md:23]: VMware Aria Operations for Applications (formerly known as Wavefront) Python SDK lets you send raw data from your Python application to Operations for Applications using a `wavefront_sender` interface. The data is then stored as metrics, histograms, and trace data. This SDK is also referred to as the Wavefront Sender SDK for Python.

File: README.md [hunk 2]
```diff
@@ -17,22 +18,22 @@
...
+# Welcome to VMware Aria Operations™ for Applications Python SDK
 
-Wavefront by VMware Python SDK lets you send raw data from your Python application to Wavefront using a `wavefront_sender` interface. The data is then stored as metrics, histograms, and trace data. This SDK is also referred to as the Wavefront Sender SDK for Python. 
+VMware Aria Operations for Applications (formerly known as Wavefront) Python SDK lets you send raw data from your Python application to Aria Operations for Applications using a `wavefront_sender` interface. The data is then stored as metrics, histograms, and trace data. This SDK is also referred to as the Wavefront Sender SDK for Python. 
 
-Although this library is mostly used by the other Wavefront Python SDKs to send data to Wavefront, you can also use this SDK directly. For example, you can send data directly from a data store or CSV file to Wavefront.
+Although this library is mostly used by the other Aria Operations for Applications Python SDKs to send data to Aria Operations for Applications, you can also use this SDK directly. For example, you can send data directly from a data store or CSV file to Aria Operations for Applications.
```
Comment (Margarita-Staneva) [inline README.md:25]: Although this library is mostly used by the other Operations for Applications Python SDKs to send data to Operations for Applications, you can also use this SDK directly. For example, you can send data directly from a data store or CSV file to Operations for Applications.

File: README.md [hunk 3]
```diff
@@ -474,14 +510,16 @@ total_failures = wavefront_sender.get_failure_count()
 
 1. Merge all the changes that need to go into the release to the master branch.
 2. Open the `setup.py` file from the top level directory of the project.
-3. Search for `version=` in the file to find the version number for example `1.8.10`.
-4. Log into Github, click **Releases** on the right, and click **Draft a new release**.
-5. For **Choose a tag**, choose the version you found in step 3, and prefix it with `v` for example `v1.8.10`. You need to enter the version where it says **Find or create new tag**.
+3. Search for `version=` in the file to find the version number for example `1.8.15`.
+4. Create a pull request, get it reviewed and approved, merge it after approval.
+5. Check [test.pypi.org](https://test.pypi.org/project/wavefront-sdk-python) for a published package, make sure it's production ready.
+6. Log into Github, click **Releases** on the right, and click **Draft a new release**.
```
Comment (Margarita-Staneva) [inline README.md:516]: Log in to GitHub, click **Releases** on the right, and click **Draft a new release**.

File: README.md [hunk 4]
```diff
@@ -474,14 +510,16 @@ total_failures = wavefront_sender.get_failure_count()
 
 1. Merge all the changes that need to go into the release to the master branch.
 2. Open the `setup.py` file from the top level directory of the project.
-3. Search for `version=` in the file to find the version number for example `1.8.10`.
-4. Log into Github, click **Releases** on the right, and click **Draft a new release**.
-5. For **Choose a tag**, choose the version you found in step 3, and prefix it with `v` for example `v1.8.10`. You need to enter the version where it says **Find or create new tag**.
+3. Search for `version=` in the file to find the version number for example `1.8.15`.
```
Comment (Margarita-Staneva) [inline README.md:513]: Search for `version=` in the file to find the version number, for example `1.8.15`.

File: README.md [hunk 5]
```diff
@@ -474,14 +510,16 @@ total_failures = wavefront_sender.get_failure_count()
...
+4. Create a pull request, get it reviewed and approved, merge it after approval.
```
Comment (Margarita-Staneva) [inline README.md:514]: 4. Create a pull request, get it reviewed and approved, and merge it after approval.

File: README.md [hunk 6]
```diff
@@ -474,14 +510,16 @@ total_failures = wavefront_sender.get_failure_count()
...
 
-6. Provide a short but descriptive title for the release.
-7. Fill in the details of the release. Please copy the markdown from the previous release and follow the same format.
-8. Click **Publish release.** to start publishing the release to pypi.org.
-9. From the Github top navigation bar of this project, click the **Actions** tab. On the first line of the list of workflows, you should see a workflow running that will publish your release to pypi.org.
-10. When the workflow from step 9 has a green checkmark next to it, go to [pypi.org](https://pypi.org/project/wavefront-sdk-python/) and verify that the latest version is published.
+8. Provide a short but descriptive title for the release.
+9. Fill in the details of the release. Please copy the markdown from the previous release and follow the same format.
+10. Click **Publish release.** to start publishing the release to pypi.org.
+11. From the Github top navigation bar of this project, click the **Actions** tab. On the first line of the list of workflows, you should see a workflow running that will publish your release to pypi.org.
```
Comment (Margarita-Staneva) [inline README.md:524]: From the GitHub top ...

**Review by Margarita-Staneva — COMMENTED** [review]
The full name is VMware Aria Operations for Applications and the short form is just Operations for Applications. 

I found some other minor things that can be fixed. :)

Btw, are you planning to update other mentions of Wavefront, for example "Wavefront service"? If this is going to happen sometime later, maybe under the first two paragraphs in the readme, you can include something like that:

Note: We're in the process of updating the product name to Operations for Applications, but in many places we still refer to it as Wavefront.

**Comment by ustinov** [comment]
> Btw, are you planning to update other mentions of Wavefront, for example "Wavefront service"? If this is going to happen sometime later, maybe under the first two paragraphs in the readme, you can include something like that:
> 
> Note: We're in the process of updating the product name to Operations for Applications, but in many places we still refer to it as Wavefront.

Thank you @Margarita-Staneva for reviewing the changes! I was actually looking for your help on how far should we go, since we're keeping the code unchanged, I'm keeping Wavefront Sender references unchanged, for example, as we still have the class named `WavefrontSender` in the code, and so on.

**Review by Margarita-Staneva — APPROVED** [review]
lgtm

**Pull request merged**